assets/scripts/forms.js

```diff
@@ -0,0 +1,26 @@
+const re = /[A-Z]/;
```

Comment:
[OPTIONAL] Always try as much as possible to use expressive and meaningful variable names.  Remember, it is very beneficial writing code that is easier for you, your future self and for someone else to look at and understand. After you’ve been programming for a while, you will begin to see that consistently formatted code is much easier for your eye to scan and quickly understand. :+1:

Reply:
Thank you for the suggestion, and the approval.